tracker/tracker_debugger_api.yaml

```diff
@@ -1318,3 +1374,912 @@ components:
+          minimum: 0
+          maximum: 65535
+          example: 11
+        t_ce:
+          description: 'Connect End - one of the Page Performance Metrics'
+          type: integer
+          minimum: 0
+          maximum: 65535
+          example: 121
+        t_qs:
+          description: 'Request Start - one of the Page Performance Metrics'
+          type: integer
+          minimum: 0
+          maximum: 65535
+          example: 121
+        t_as:
+          description: 'Response Start - one of the Page Performance Metrics'
+          type: integer
+          minimum: 0
+          maximum: 65535
+          example: 557
+        t_ae:
+          description: 'Response End - one of the Page Performance Metrics'
+          type: integer
+          minimum: 0
+          maximum: 65535
+          example: 569
+        t_dl:
+          description: 'DOM Loading - one of the Page Performance Metrics'
+          type: integer
+          minimum: 0
+          maximum: 65535
+          example: 579
+        t_di:
+          description: 'DOM Interactive - one of the Page Performance Metrics'
+          type: integer
+          minimum: 0
+          maximum: 4294967295
+          example: 974
+        t_ls:
+          description: 'DOM Content Loaded Event Start - one of the Page Performance Metrics'
+          type: integer
+          minimum: 0
+          maximum: 4294967295
+          example: 974
+        t_le:
+          description: 'DOM Content Loaded Event End - one of the Page Performance Metrics'
+          type: integer
+          minimum: 0
+          maximum: 4294967295
+          example: 975
+        t_dc:
+          description: 'DOM Complete - one of the Page Performance Metrics'
+          type: integer
+          minimum: 0
+          maximum: 4294967295
+          example: 3822
+        t_ee:
+          description: 'Load Event End - one of the Page Performance Metrics'
+          type: integer
+          minimum: 0
+          maximum: 4294967295
+          example: 3923
+        _rcn:
+          description: 'The Campaign name'
+          type: string
+          example: 'deprecated campaign name parameter'
+        _rck:
+          description: 'The Campaign Keyword'
+          type: string
+          example: 'deprecated keyword parameter'
+        dimension1:
+          description: Custom dimension 1
+          type: string
+          example: 'custom_dimension_1_value'
+        dimension2:
+          description: Custom dimension 2
+          type: string
+          example: 'custom_dimension_2_value'
+        dimension3:
+          description: Custom dimension 3
+          type: string
+          example: 'custom_dimension_3_value'
+        dimension4:
+          description: Custom dimension 4
+          type: string
+          example: 'custom_dimension_4_value'
+        dimension5:
+          description: Custom dimension 5
+          type: string
+          example: 'custom_dimension_5_value'
+        dimension6:
```

Comment:
We should stop at 3 or 5. 100 is unnecessary noise.

Reply:
I didn't find any "and so on up till X" option in OpenApi - and since we either document every possible parameter or we don't I decided to document those as well.

Reply:
It produces noise - no doubt about it.
But removing them seems unfair.